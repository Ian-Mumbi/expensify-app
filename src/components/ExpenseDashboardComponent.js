import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpenseSummaryComponent from './ExpenseSummaryComponent'

const ExpenseDashboardComponent = (props) => (
        <div>
          <ExpenseSummaryComponent />
          <ExpenseListFilters />
          <ExpenseList />
        </div>
)

export default ExpenseDashboardComponent