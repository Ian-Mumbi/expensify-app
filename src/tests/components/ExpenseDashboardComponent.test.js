import React from 'react'
import { shallow } from 'enzyme'
import ExpenseDashboardComponent from '../../components/ExpenseDashboardComponent'

test('should render ExpenseDashboardComponent correctly', () => {
    const wrapper = shallow(<ExpenseDashboardComponent />)
    expect(wrapper).toMatchSnapshot()
})