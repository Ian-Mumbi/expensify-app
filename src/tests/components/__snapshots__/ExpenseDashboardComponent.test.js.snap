// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardComponent correctly 1`] = `
<div>
  <Connect(ExpenseSummaryComponent) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
